index.js

```diff
@@ -1,11 +1,19 @@
 const express = require('express');
 const db_conn = require('./db_connection');
+// adding code here
+const bodyParser = require('body-parser');
+const userRoutes = require('./userRoutes');
```

Comment:
can you please merge/pull the dev branch to your branch so you can get the userRoutes.js and remove the merge conflict

Reply:
done